dev/docker/middleware.entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+# cleanup filesForUpload dir before start
+rm -rf /uploads/*
+# populate shared volume with the files
+cp -r filesForUpload/* /uploads/
+# start the middleware service
+yarn start
```

Comment:
I think we could move this to the middleware container itself

Reply:
the `/uploads` folder directly depends on the compose file living in `web` and volumes used by the container. if we change the volume path in the `docker-compose` file, we've to come to the middleware repo just to fix the script. should we still move this?

Reply:
the `/uploads` folder directly depends on the compose file living in `web` and volumes used by the container. if we change the volume path in the `docker-compose` file, we've to come to the middleware repo just to fix the script. should we still move this?